src/server/server_impl.go

```diff
@@ -88,6 +96,14 @@ func NewJsonHandler(svc pb.RateLimitServiceServer) func(http.ResponseWriter, *ht
 			return
 		}
 
+		// Generate trace
+		_, span := tracer.Start(context.TODO(), "NewJsonHandler Remaining Execution",
```

Comment:
Without a real context is there any real use for this?

Reply:
I have confusion on this part too. The NewJsonHandler for http endpoint calls the `ShouldRateLimit` without passing a context. Should we initiate a background context in the handler and pass it to both `ShouldRateLimit` function and the tracer for correctly handling the context?